src/frontend/screens/Audio/CreateRecording/index.tsx

```diff
@@ -26,26 +28,43 @@ export function CreateRecording() {
     }
   }, [recordingState]);
 
-  if (recordingState === 'idle') {
-    return <RecordingIdle onPressRecord={startRecording} />;
-  }
-
-  if (recordingState === 'active') {
-    return (
-      <RecordingActive
-        duration={status?.durationMillis || 0}
-        onPressStop={stopRecording}
-      />
-    );
-  }
-
-  if (recordingState === 'done') {
-    return (
-      <RecordingDone
-        uri={uri || ''}
-        duration={status?.durationMillis || 0}
-        reset={reset}
+  return (
+    <>
+      {recordingState === 'idle' && (
+        <RecordingIdle onPressRecord={startRecording} />
+      )}
+      {recordingState === 'active' && (
+        <RecordingActive
+          duration={status?.durationMillis || 0}
+          onPressStop={stopRecording}
+        />
+      )}
+      {recordingState === 'done' && (
+        <RecordingDone
+          uri={uri || ''}
+          duration={status?.durationMillis || 0}
+          reset={reset}
+        />
+      )}
+      <ErrorBottomSheet
+        error={error}
```

Comment:
I think we should throw an error if the `recordingState` is done and there is no uri:

```ts
if(recordingState === 'done' && !uri){
    setError(new Erorr('no uri'))
}

return (...rest)
```